docs/design/cloud_infra.md

```diff
@@ -0,0 +1,55 @@
+# Cloud Infrastructure
+
+The entire infrastructure of the project will be hosted on GCP
+
+- Secrets: GCP Secrets Manager
+  - Manually store sensitive information
+- Kubernetes Cluster: GKE (Google Kubernetes Engine)
+- Images: GCR (Google Container Registry)
+
```

Comment:
Can you recommend a plan for hosting our database (MongoDB)? Do you think we should host our own instance on a VM or should we use Atlas?